scripts/conflicts/package-lock.json

```diff
@@ -1,8 +1,361 @@
 {
   "name": "conflicts",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
I think you've inadvertently pushed changes to this file, because you'r using npm 7.x+ 
Can you please rollback these changes and use npm 6.x until we officially support newer versions?